modules/tensor_mechanics/src/materials/ComputeInterfaceStress.C

```diff
@@ -18,51 +18,75 @@ validParams<ComputeInterfaceStress>()
   params.addClassDescription(
       "Stress in the plane of an interface defined by the gradient of an order parameter");
   params.addCoupledVar("v",
-                       "Order parameter that defines the interface. The interface is the region "
+                       "Order parameters that defines the interface. The interface is the region "
                        "where the gradient of this order parameter is non-zero.");
-  params.addRequiredParam<Real>("stress", "Planar stress");
-  params.addRangeCheckedParam<Real>("op_range",
-                                    1.0,
-                                    "op_range > 0.0",
-                                    "Range over which order parameters change across an "
-                                    "interface. By default order parameters are assumed to "
-                                    "vary from 0 to 1");
+  params.addRequiredParam<std::vector<Real>>("stress", "Planar stress");
```

Comment:
Since you are allowing either a single stress or a different stress for each OP (as in the error check below) I think there should be some mention of this in the help text here

Reply:
Seconding Larry's comment here.